Classes/Controller/BackendController.php

```diff
@@ -101,6 +101,13 @@ class BackendController extends ActionController
      */
     protected $styleAndJavascriptInclusionService;
 
+    /**
+     * @Flow\Inject
+     * @var ClipboardStateContainer
+     */
+    protected $clipboardStateContainer;
```

Comment:
It's just a (node) clipboard

    /**
     * @Flow\Inject
     * @var NodeClipboardInterface
     */
    protected $clipboard;

could work, no?